ghcide.cabal

```diff
@@ -161,6 +162,9 @@ library
         Development.IDE.Plugin
         Development.IDE.Plugin.Completions
         Development.IDE.Plugin.CodeAction
+        Development.IDE.Plugin.Formatter
```

Comment:
Why do these end up in Ghcide? I guess they leave in a future diff?

Reply:
`Formatter` contains machinery needed to convert an HLS `FormattingProvider` into a `PartialHandler` for ghcide